cli/release.go

```diff
@@ -276,12 +277,17 @@ func getReleases(releaseURL string, includeAll bool) ([]semver.Version, map[stri
 
 	releasesArray := []map[string]interface{}{}
 
-	bytes, err := ioutil.ReadAll(resp.Body)
+	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
 		return nil, nil, fmt.Errorf("failed to read release details: %v", err)
+	} else if resp.StatusCode == http.StatusForbidden && bytes.Contains(body, []byte("limit exceeded")) {
+		// Do not fail if rate limit is exceeded
+		kataLog.WithField("url", releaseURL).
+			Warn("API rate limit exceeded. Try again later. Read https://developer.github.com/v3/#rate-limiting for more information")
```

Comment:
I'm not 100% happy about hard-coding this URL (including the "v3", but given we want to be as helpful to users as possible, I guess we'll just have to maintain this (or hope GH auto-redirect if they ever remove "v3" entirely).